### PR TITLE
VideoCommon: Minor changes

### PR DESCRIPTION
**Pull request by lioncash**
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/3198)

### CODE REVIEW

File: Source/Core/VideoCommon/RenderBase.cpp [hunk 1]
```diff
@@ -222,11 +222,10 @@ bool Renderer::CalculateTargetSize(unsigned int framebuffer_width, unsigned int
 			efb_scale_denominatorX = efb_scale_denominatorY = 1;
 
 
-			int maxSize;
-			maxSize = GetMaxTextureSize();
-			if ((unsigned)maxSize < EFB_WIDTH * efb_scale_numeratorX / efb_scale_denominatorX)
+			const unsigned int max_size = static_cast<unsigned int>(GetMaxTextureSize());
```
Comment (degasus) [inline Source/Core/VideoCommon/RenderBase.cpp]: I think function is only used here, so just change the API to return an unsigned integer.

**Comment by degasus** [comment]
any updates?

**Comment by bb010g** [comment]
ping

**Comment by Fallcrest** [comment]
@lioncash Rebase? Cleaning up VideoCommons will be good, as the Vulkan backend is being worked on and a Metal backend is on the way from either @OatmealDome or @WilliamLCobb (in conjunction with the work on the iOS build).

**Comment by RisingFog** [comment]
This seems to be a rather simple cleanup, is there any reason for the long holdup?

**Pull request closed without merging**